index.js

```diff
@@ -0,0 +1,123 @@
+webix.ready(function () {
+  var head = {
```

Comment:
const or let are used instead of var now

Reply:
> const or let are used instead of var now

Yes, I know) just folliwed the webix guide, thought that webix uses var. Ok, I see